src/components/buttons/Close/Close.tsx

```diff
@@ -16,18 +16,26 @@ export default class Close extends React.Component<IProps> {
 		position: 'absolute',
 	};
 
+	onKeyDown = (event: any) => {
+		if (event.key === 'Enter'){
+			event.target.click();
+		}
+	}
+
 	render () {
 		return (
 			<span
+				tabIndex={0}
```

Comment:
On second thought, I wonder if we want to add an optional prop for this and, if undefined, don't set `tabIndex` at all. Put it on the implementation. Plus, there are instances where using no tabIndex might be desirable here.

Reply:
So, the MDN docs strongly advise against setting tab indices greater than 1 in favor of organizing markup in the order you want the tabs to go. So, instead of messing with tabIndex props for all, I went with the following solutions. Let me know if you disagree with the approach: 

1. For the RadioBlock, I set the tabIndex to -1 when the item is selected, that way the focus goes immediately to the next selectable option.

2. For the buttons, I changed the order of the JSX for the 'Go Back' buttons and 'Continue' buttons, which gave the desired tab order of Form inputs first, then 'Continue', then 'Go Back'.

Reply:
Awesome! I was wondering if we could reorder those elements and didn't know how feasible that'd be. Looks like you pulled it off. Nice work!